dogs.html

```diff
@@ -0,0 +1,167 @@
+        <p class="dogs-text">Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+        <button onclick="dogAdopt('122.46')" class="adopt-button">Adopt</button>
+      </div>
+
+      <div class= "box">
+        <img onclick="dogInfo('Jack','Beagle', '$122.46')" src="images/jack-card.jpg" alt="Dog named Jack who is small and has brown and white fur">
+        <h3>Jack</h3>
+        <p class="dogs-cost-text"><b>Cost to Adopt:</b></p>
+        <p class="cost">$122.46</p>
+        <p class="dogs-text">Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+        <button onclick="dogAdopt('122.46')" class="adopt-button">Adopt</button>
+      </div>
+
+      <div class= "box">
+        <img onclick="dogInfo('Duffy','Pitbull', '$122.46')" src="images/duffy-card.jpg" alt="Dog named Duffy who is medium-sized and has white and brown fur">
+        <h3>Duffy</h3>
+        <p class="dogs-cost-text"><b>Cost to Adopt:</b></p>
+        <p class="cost">$122.46</p>
+        <p class="dogs-text">Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+        <button onclick="dogAdopt('122.46')" class="adopt-button">Adopt</button>
+      </div>
+
+      <div class= "box">
+        <img onclick="dogInfo('Lucas','Black Laborador Mix', '$122.46')" src="images/lucas-card.jpg" alt="Dog named Lucas who is large and has black fur">
+        <h3>Lucas</h3>
+        <p class="dogs-cost-text"><b>Cost to Adopt:</b></p>
+        <p class="cost">$122.46</p>
+        <p class="dogs-text">Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+        <button onclick="dogAdopt('122.46')" class="adopt-button">Adopt</button>
+      </div>
+
+      <div class= "box">
+        <img onclick="dogInfo('Jake','Laborador', '$122.46')" src="images/jake-card.jpg" alt="Dog named Jake who is large and has tan fur">
+        <h3>Jake</h3>
+        <p class="dogs-cost-text"><b>Cost to Adopt:</b></p>
+        <p class="cost">$122.46</p>
+        <p class="dogs-text">Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+        <button onclick="dogAdopt('122.46')" class="adopt-button">Adopt</button>
+      </div>
+
+      <div class= "box">
+        <img onclick="dogInfo('Angus','Boxer', '$122.46')" src="images/angus-card.jpg" alt="Dog named Angus who is medium-sized and has brown fur">
+        <h3>Angus</h3>
+        <p class="dogs-cost-text"><b>Cost to Adopt:</b></p>
+        <p class="cost">$122.46</p>
+        <p class="dogs-text">Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+        <button onclick="dogAdopt('122.46')" class="adopt-button">Adopt</button>
```

Comment:
click adopt and view the web console. you are still referencing "dogAdopt()" which does not exist. you can remove each of these because your jQuery is referencing adopt-button class instead.

Reply:
Thanks, Tyler! I thought my code would automatically updated on dogs.html, but that was, again, another mistake.